src/containers/ruby-tab.jsx

```diff
@@ -18,6 +18,9 @@ const RubyTab = ({rubyCode}) => (
             useSoftTabs: true,
             showInvisibles: true
         }}
+        style={{
+            fontFamily: 'monaco'
```

Comment:
AceEditorのデフォルトを確認したら、以下のようになっていましたので、修正をお願いします。

```suggestion
            fontFamily: 'Monaco', 'Menlo', 'Ubuntu Mono', 'Consolas', 'source-code-pro', 'monospace'
```

Reply:
最後の 'monospace' は実際には monospace というようにシングルクォートでくくられて *いなかった* のですが、eslint でエラーになりそうなので、くくっています。

Reply:
デフォルトで文字が被るのでデフォルトを指定し直しても、文字被りが発生してしまいます。

Reply:
あぁ、それはそうか。Manacoにはない日本語フォントが問題になっていそうだね。
'Menlo'が悪さをしているのかも。
ようは、まともな日本語フォントを追加しないといけないってことだね。

ちなみに、
http://ajaxorg.github.io/ace-builds/editor.html
で確認するとどうかな？やはり文字がかぶりますか？

もし、文字がかぶらないならば、 scratch-gui + react-ace の組み合わせに問題があるといえます。

Reply:
被ります。
因みに、悪さしているのは"Ubuntu Nono"みたいです。

Reply:
現状、Ubuntu上で文字被りが発生しますので、
因みに余談になりますが、現行のスモウルビー（無印）でもUbuntu上だと文字被りが発生します。

Reply:
それでは、ワークアラウンドとして、 'Ubuntu Mono' だけを省いてもらえますかね。
それで問題なければ、マージします。

Reply:
修正しました。